### DISCUSSION TITLE
Spotify integration

### DISCUSSION THREAD

**Discussion by poltrian** [navidrome/navidrome#2477] [Q&A]
I put the Spotify id and the secret in my config toml as in the docs shown.  But in the log I can see, it's missing. What am I doing wrong?

**Comment by metalheim** [comment]
Just to be sure, are you certain you have the right syntax for toml?
Here is an example of the syntax that should work:
https://github.com/navidrome/navidrome/discussions/2469#discussioncomment-6658678
